spec/features/orders/creation_spec.rb

```diff
@@ -46,8 +46,10 @@
 
       new_order = Order.last
 
+      visit("/orders/#{new_order.id}")
```

Comment:
I just realized if we are visiting the path, we don't have to expect it... I think... because visiting it means it works